src/fs/driver/fat/fat_common.c

```diff
@@ -0,0 +1,1608 @@
+/**
+ * @file   fat_common.c
+ * @brief  VFS-independent part of FAT driver
+ * @data   9 Apr 2015
+ * @author Denis Deryugin
+ */
+
+#include <assert.h>
+#include <errno.h>
+#include <fcntl.h>
+#include <limits.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include <embox/block_dev.h>
+#include <framework/mod/options.h>
+#include <fs/fat.h>
+#include <mem/misc/pool.h>
+#include <util/math.h>
+
+#define LABEL "EMBOX_DISK"
+#define SYSTEM "FAT12"
+
+extern uint8_t fat_sector_buff[512]; /* XXX */
```

Comment:
Early I commented this variable, except that comment here is a constant 512 although first was 
uint8_t fat_sector_buff[FAT_MAX_SECTOR_SIZE];